### PR TITLE
Magazine pouches are capable of tac reload through right click.

### PR DESCRIPTION
**Pull request by Xander3359**
## About The Pull Request
Makes it so you can tac-reload your gun by right clicking a magazine pouch.
## Why It's Good For The Game
more QOL.
## Changelog
:cl:
qol: You can right click a magazine pouch with your gun to tac-reload it.
/:cl:

### CODE REVIEW

**Review by Lumipharon — CHANGES_REQUESTED** [review]
You're copy pasting functionality across different types of storage. Just make it a configurable thing at the storage level (or potentially even a component so it can work with storage attachments)

**Comment by Al-1ce** [comment]
as someone who doesnt use right click this is cool and must be speedmerged

File: code/game/objects/items/storage/pouch.dm [hunk 1]
```diff
@@ -211,6 +211,24 @@
...
+		if(user.l_hand && user.r_hand || length(gun_to_reload.chamber_items))
+			gun_to_reload.tactical_reload(item_to_reload_with, user)
+		else
+			gun_to_reload.reload(item_to_reload_with, user)
+		orient2hud()
+		return
+
+/obj/item/storage/pouch/magazine/examine(mob/user)
+	. = ..()
+	. += span_notice("To perform a reload with the amunition inside, right click on the belt with any compatible gun.")
```
Comment (ForeverNumbered) [inline code/game/objects/items/storage/pouch.dm:230]: misspelled ammunition

**Comment by ForeverNumbered** [comment]
can a delay to the action be added such that there is a cost to reloading this way

will this give the marine an ability to reload if they are missing the other arm?

**Pull request closed without merging**